src/components/tagsInput/index.js

```diff
@@ -236,24 +239,48 @@ export default class TagsInput extends BaseComponent {
     return _.get(item, 'label');
   }
 
-  renderLabel(tag, shouldMarkTag) {
+  checkTagValidate(tag, validationErrorMessage) {
+    return validationErrorMessage ? tag.invalid : false;
+  }
+
+  renderLabel(tag, shouldMarkTag, isTagNotValidate) {
     const typography = this.extractTypographyValue();
     const label = this.getLabel(tag);
 
     return (
       <View row centerV>
-        {shouldMarkTag && <Image style={styles.removeIcon} source={Assets.icons.x}/>}
-        <Text style={[styles.tagLabel, typography]} accessibilityLabel={`${label} tag`}>
-          {shouldMarkTag ? 'Remove' : label}
+        {shouldMarkTag && (
+          <Image
+            style={[isTagNotValidate ? styles.inValidTagRemoveIcon : styles.removeIcon]}
+            source={Assets.icons.x}
+          />)
+        }
+        <Text
+          style={[isTagNotValidate ? (shouldMarkTag ? styles.inValidMarkedTagLabel : styles.inValidTagLabel)
+            : styles.tagLabel, typography]} accessibilityLabel={`${label} tag`}
+        >
+          {isTagNotValidate ? label : (shouldMarkTag ? 'Remove' : label)}
```

Comment:
you can simplify the ternary condition
```suggestion
          {shouldMarkTag && !isTagNotValidate ? 'Remove' : label}
```

Reply:
done